rocm_agent_enumerator

```diff
@@ -135,8 +135,8 @@ def readFromROCMINFO(search_arch_name = False):
         break
       # run rocminfo
       rocminfo_output = subprocess.Popen(rocminfo_executable, stdout=subprocess.PIPE).communicate()[0].decode("utf-8").split('\n')
-      term1 = re.compile("Cannot allocate memory")
-      term2 = re.compile("HSA_STATUS_ERROR_OUT_OF_RESOURCES")
+      term1 = re.compile(r"Cannot allocate memory")
+      term2 = re.compile(r"HSA_STATUS_ERROR_OUT_OF_RESOURCES")
```

Comment:
There is no practical difference between using a raw string and a non-raw string, as there are no backslash escape characters involved here.